.circleci/config.yml

```diff
@@ -0,0 +1,55 @@
+version: 2.1
+
+#Orb importing
+orbs:
+  aws-cli: circleci/aws-cli@1.4.0
+
+jobs:
+
+  build:
+    docker:
+      - image: circleci/node:12
+    steps:
+      - checkout
+      - run: npm install
+      - run: npm run build
+      - persist_to_workspace:
+          root: dist
+          paths:
+            - ?*
+
+  push_to_cdn:
+    executor: aws-cli/default
+    steps:
+      - checkout
+      - attach_workspace:
+          at: dist
+      - aws-cli/setup:
+          aws-access-key-id: AWS_ACCESS_KEY_ID
+          aws-secret-access-key: AWS_SECRET_ACCESS_KEY
+          aws-region: AWS_REGION
+          profile-name: default
+      - run: 
+          name: push
+          command: |
+            # get version from code? card or cards?
```

Comment:
Are these questions we still need to address?

Reply:
Oh good point @dominicsardina . I guess: 
(1) getting the version from `package.json` is safe
(2) `/sdk/card` or `/sdk/cards`?

Reply:
The repo and npm package are "cards". Am I using them interchangeably somewhere? I can help standardize on "cards" if I used "card" somewhere.

Reply:
All good. Let's use `/sdk/cards`.